src/azure-cli/azure/cli/command_modules/lab/tests/latest/recordings/test_lab_vm.yaml

```diff
@@ -2339,11 +2339,10 @@ interactions:
       accept-language:
       - en-US
     method: GET
-    uri: https://graph.windows.net/00000000-0000-0000-0000-000000000000/me?api-version=1.6
+    uri: https://graph.microsoft.com/v1.0/me
```

Comment:
I don't know why `test_lab_vm` is marked as `record_only`:

https://github.com/Azure/azure-cli/blob/fd5779f74f78aad16d56c67a1644d4299a5904ff/src/azure-cli/azure/cli/command_modules/lab/tests/latest/test_lab.py#L19-L20

so I manually edited the recording file.